doaj/DOAJ/.ipynb_checkpoints/DOAJ_test-checkpoint.ipynb

```diff
@@ -0,0 +1,291 @@
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "list"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# data_list\n",
+    "type(pissn_list)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dict2=db.getAPC(pissn_list[:500])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'1594-4077': {'currency': 'USD', 'average_price': 900},\n",
+       " '0032-5791': {'currency': 'USD', 'average_price': 2000},\n",
+       " '0036-4665': {'currency': 'USD', 'average_price': 400},\n",
+       " '1475-2875': {'currency': 'GBP', 'average_price': 1430},\n",
+       " '0213-9111': {'currency': 'EUR', 'average_price': 450}}"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dict2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dict3= db.convert_apc2usd(dict2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'1594-4077': {'currency': 'USD', 'average_price': 900},\n",
+       " '0032-5791': {'currency': 'USD', 'average_price': 2000},\n",
+       " '0036-4665': {'currency': 'USD', 'average_price': 400},\n",
+       " '1475-2875': {'currency': 'USD', 'average_price': 1790},\n",
+       " '0213-9111': {'currency': 'USD', 'average_price': 508}}"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dict3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'1594-4077': {'currency': 'USD', 'average_price': 900},\n",
+       " '0032-5791': {'currency': 'USD', 'average_price': 2000},\n",
+       " '0036-4665': {'currency': 'USD', 'average_price': 400},\n",
+       " '1475-2875': {'currency': 'GBP', 'average_price': 1430},\n",
+       " '0213-9111': {'currency': 'EUR', 'average_price': 450}}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
the folder and file .ipynb_checkpoints/DOAJ_test-checkpoint.ipynb should not be in the repo